CHANGELOG.md

```diff
@@ -15,6 +15,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - label `thread` for per thread reporting net statistics metrics.
 
+### Deprecated
```

Comment:
Please add section "Added" with new metrics
And log message like this: https://github.com/tarantool/metrics/blob/c3aeda62bbeb12f6b4506620d635bada742af6ae/metrics/init.lua#L129